Modix.Bot/Behaviors/ModerationLoggingBehavior.cs

```diff
@@ -109,10 +109,10 @@ private string GetBatchUrl(long? batchId)
         private static readonly Dictionary<(ModerationActionType, InfractionType?), string> _renderTemplates
             = new Dictionary<(ModerationActionType, InfractionType?), string>()
             {
-                { (ModerationActionType.InfractionCreated,   InfractionType.Notice),  "`[{0}]` **{1}** recorded the following note for **{3}** (`{4}`) ```\n{5}```" },
-                { (ModerationActionType.InfractionCreated,   InfractionType.Warning), "`[{0}]` **{1}** recorded the following warning for **{3}** (`{4}`) ```\n{5}```" },
-                { (ModerationActionType.InfractionCreated,   InfractionType.Mute),    "`[{0}]` **{1}** muted **{3}** (`{4}`) for reason ```\n{5}```" },
-                { (ModerationActionType.InfractionCreated,   InfractionType.Ban),     "`[{0}]` **{1}** banned **{3}** (`{4}`) for reason ```\n{5}```" },
+                { (ModerationActionType.InfractionCreated,   InfractionType.Notice),  "`[{0}] [{2}]` **{1}** recorded the following note for **{3}** (`{4}`) ```\n{5}```" },
```

Comment:
This is me just being me - but maybe we should stay consistent with the way we structure this.

In the updates, we show the Id after the type, i.e. `deleted a notice (ID)`

So this should possibly read: `[{0}] ***{1}*** recorded the following note ({2}) for ***{3}*** ({4})`

Reply:
I did consider that, but if we do the same thing for ban/mutes, those would look like: `[11:59:59] Username#1234 banned (123) Username#2345 (1111111111111111) for reason`..., and I wasn't sure if I like a bunch of numbers being right next to each other without clear explanation on which is which.

Currently Vortex basically does the same thing, where the infraction ID goes in front, right after the timestamp.

Reply:
I updated all the log messages to have the infraction IDs in the front and not anywhere else - does this look okay?

Please do note that I haven't actually built / tested the code - I'm editing this on the web interface, so while I don't see any obvious errors it would be nice to be able to use some extra check from you folks.